Packs/Netskope/Integrations/NetskopeAPIv2/NetskopeAPIv2.yml

```diff
@@ -0,0 +1,1050 @@
+category: Network Security
+commonfields:
+  id: netskope_api_v2
+  version: -1
+configuration:
+- defaultvalue: https://{environment}.{region}.goskope.com
+  display: Server URL
+  name: url
+  type: 0
+  required: true
+- defaultvalue: 'false'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- additionalinfo: "Netskope API access token (make sure to generate token for the following endpoints: api/v2/events/data/application, api/v2/events/data/audit, api/v2/events/data/page, api/v2/events/data/network, api/v2/events/data/infrastructure, api/v2/events/data/alert, api/v2/policy/urllist (read + write), api/v2/policy/urllist/deploy (read + write), api/v2/scim/Users)."
```

Comment:
this is quite a lot of info, perhaps it's better to point to the integration readme? @meital-aizen